Sources/Views/AvatarView.swift

```diff
@@ -24,27 +24,25 @@
 
 import Foundation
 
-open class AvatarView: UIView {
+open class AvatarView: UIImageView {
 
     // MARK: - Properties
-
-    open var avatar: Avatar = Avatar()
-
-    open var imageView = UIImageView()
-
-    public var image: UIImage? {
-        return imageView.image
+    
+    open var initials: String? {
```

Comment:
Do you think we should just make `setImageFrom(initials:)` public?

Reply:
The reason I say that is because the property will store a reference to `initials`... then if we use `set(avatar: Avatar)` a different image may be set... but `initials` property could hold the wrong value.

Reply:
ok, I'll make `initials` to `private`.